integration-tests/features/environment.py

```diff
@@ -501,6 +511,7 @@ def before_all(context):
     context.wait_for_jobs_debug_api_service = _wait_for_jobs_debug_api_service
     context.wait_for_component_search_service = _wait_for_component_search_service
     context.is_3scale_staging_running = _is_3scale_staging_running
+    context.is_3scale_preview_running = _is_3scale_preview_running
```

Comment:
why we need to distinguish between staging and preview? IMHO it is not necessarry